src/material/schematics/ng-generate/m3-theme/index.spec.ts

```diff
@@ -0,0 +1,493 @@
+      [10, '#3e001d'],
+      [20, '#5e1133'],
+      [25, '#6c1d3e'],
+      [30, '#7b2949'],
+      [35, '#893455'],
+      [40, '#984061'],
+      [50, '#b6587a'],
+      [60, '#d57194'],
+      [70, '#f48bae'],
+      [80, '#ffb0c8'],
+      [90, '#ffd9e2'],
+      [95, '#ffecf0'],
+      [98, '#fff8f8'],
+      [99, '#fffbff'],
+      [100, '#ffffff'],
+    ]),
+  ],
+  [
+    'secondary',
+    new Map<number, string>([
+      [0, '#000000'],
+      [10, '#2b151c'],
+      [20, '#422931'],
+      [25, '#4e343c'],
+      [30, '#5a3f47'],
+      [35, '#674b53'],
+      [40, '#74565f'],
+      [50, '#8e6f77'],
+      [60, '#aa8891'],
+      [70, '#c6a2ab'],
+      [80, '#e2bdc6'],
+      [90, '#ffd9e2'],
+      [95, '#ffecf0'],
+      [98, '#fff8f8'],
+      [99, '#fffbff'],
+      [100, '#ffffff'],
+    ]),
+  ],
+  [
+    'tertiary',
+    new Map<number, string>([
+      [0, '#000000'],
+      [10, '#2e1500'],
+      [20, '#48290c'],
+      [25, '#543416'],
+      [30, '#623f20'],
+      [35, '#6f4a2a'],
+      [40, '#7c5635'],
+      [50, '#986e4b'],
+      [60, '#b48862'],
+      [70, '#d1a27b'],
+      [80, '#efbd94'],
+      [90, '#ffdcc2'],
+      [95, '#ffeee2'],
+      [98, '#fff8f5'],
+      [99, '#fffbff'],
+      [100, '#ffffff'],
+    ]),
+  ],
+  [
+    'neutral',
+    new Map<number, string>([
+      [0, '#000000'],
+      [10, '#201a1b'],
+      [20, '#352f30'],
+      [25, '#413a3b'],
+      [30, '#4c4546'],
+      [35, '#585052'],
+      [40, '#645c5e'],
+      [50, '#7e7576'],
+      [60, '#988e90'],
+      [70, '#b3a9aa'],
+      [80, '#cfc4c5'],
+      [90, '#ebe0e1'],
+      [95, '#faeeef'],
+      [98, '#fff8f8'],
+      [99, '#fffbff'],
+      [100, '#ffffff'],
+      [4, '#120d0e'],
+      [6, '#171213'],
+      [12, '#241e1f'],
+      [17, '#2f282a'],
+      [22, '#3a3334'],
+      [24, '#3e3739'],
+      [87, '#e3d7d9'],
+      [92, '#f1e5e7'],
+      [94, '#f7ebec'],
+      [96, '#fdf1f2'],
+    ]),
+  ],
+  [
+    'neutral-variant',
+    new Map<number, string>([
+      [0, '#000000'],
+      [10, '#24191c'],
+      [20, '#3a2d30'],
+      [25, '#45383b'],
+      [30, '#514347'],
+      [35, '#5d4f52'],
+      [40, '#6a5b5e'],
+      [50, '#837377'],
+      [60, '#9e8c90'],
+      [70, '#b9a7ab'],
+      [80, '#d5c2c6'],
+      [90, '#f2dde2'],
+      [95, '#ffecf0'],
+      [98, '#fff8f8'],
+      [99, '#fffbff'],
+      [100, '#ffffff'],
+    ]),
+  ],
+  [
+    'error',
+    new Map<number, string>([
+      [0, '#000000'],
+      [10, '#410002'],
+      [20, '#690005'],
+      [25, '#7e0007'],
+      [30, '#93000a'],
+      [35, '#a80710'],
+      [40, '#ba1a1a'],
+      [50, '#de3730'],
+      [60, '#ff5449'],
+      [70, '#ff897d'],
+      [80, '#ffb4ab'],
+      [90, '#ffdad6'],
+      [95, '#ffedea'],
+      [98, '#fff8f7'],
+      [99, '#fffbff'],
+      [100, '#ffffff'],
+    ]),
+  ],
+]);
+
+const testM3ThemeScss = `@use 'sass:map';
+@use '@angular/material-experimental' as matx;
+
+$_palettes: (
```

Comment:
Is it possible to leave a comment with the seed colors here, so users can remember what they inputed